pkg/k8s/watchers/endpointsynchronizer.go

```diff
@@ -109,6 +109,11 @@ func (epSync *EndpointSynchronizer) RunK8sCiliumEndpointSync(e *endpoint.Endpoin
 					return nil
 				}
 
+				if !e.NamedPortsSet() {
+					scopedLog.Debug("Skipping CiliumEndpoint update because does not have k8s ports set")
```

Comment:
This message should be something like "Skipping CiliumEndpoint update because k8s metadata is not yet available". Many EPs have no named ports, after all.